codegen-server/src/main/kotlin/software/amazon/smithy/rust/codegen/server/smithy/protocols/ServerHttpProtocolGenerator.kt

```diff
@@ -1075,11 +1059,11 @@ private class ServerHttpProtocolImplGenerator(
             }
         }
     }
-    private fun getStreamingBodyTraitBounds(operationShape: OperationShape): String {
+
+    private fun streamingBodyTraitBounds(operationShape: OperationShape) =
```

Comment:
TIL: if you use "=" instead of a function with brackets, you can avoid to fill the return type.. Interesting.

Reply:
It's only considered idiomatic for one-liner functions. You avoid the return type and the `return` keyword, you can just use an expression.